contrib/mongodb/mongo-go-driver/mongo/mongo.go

```diff
@@ -29,13 +30,12 @@ type monitor struct {
 
 func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 	hostname, port := peerInfo(evt)
-	statement := evt.Command.ToExtJSON(false)
-
+	b, _ := bson.MarshalExtJSON(evt.Command, true, false)
```

Comment:
@dsmontoya The "canonical" value was previously `false`. I'm not sure what its purpose is but I see you've changed it to `true`. Can you please explain why?

Reply:
I'm pretty sure is a mistake, because I tried to keep the original intended behavior.